uint16 navboard_seq
float32[3] acceleration
float32[3] gyro
float32 height_us
byte us_initialized
byte height_fresh
float32[3] orientation_mag

